dspace-spring-rest/src/test/java/org/dspace/app/rest/SubmissionDefinitionsControllerIT.java

```diff
@@ -95,8 +95,8 @@ public void findByCollection() throws Exception {
     public void findCollections() throws Exception {
         //Match only that a section exists with a submission configuration behind
         getClient().perform(get("/api/config/submissiondefinitions/traditional/collections"))
-                   //The status has to be 200 OK
-                   .andExpect(status().isOk());
+                   //Does this status change make sense?
```

Comment:
yes and no, acording to the rest contract https://github.com/DSpace/Rest7Contract/blob/master/submissiondefinitions.md#collections
as there is no explicit mapping toward the traditional submissionform an empty list is returned...
I'm not sure if no content is the right return here and more in general where list are expected... maybe a zero size page is a better option?